README.md

```diff
@@ -1,97 +1,167 @@
-# mxGraph
+# maxGraph
 
-## Getting started
+<!-- next to come: npm and releases badge
+[![npm version](https://img.shields.io/npm/v/xxx.svg?style=flat)](https://www.npmjs.com/package/xxx)
+-->
+[![build status](https://github.com/maxGraph/maxGraph/workflows/Build/badge.svg)](https://github.com/maxGraph/maxGraph/actions/workflows/build.yml)
 
-### Setting up local development environment
+maxGraph is a TypeScript library which can display and allow interaction with vector diagrams. At a high level, it provides: 
+- **Nodes**, also known as **vertices** which are typically represented by shapes like rectangles.
+- **Edges** which can be lines and arrows which normally point between one node and another.
 
-In the project root directory, execute
+It provides many of the diagramming features which would be expected by a piece of presentation software like Microsoft® PowerPoint™ or LibreOffice® Impress such as being able to resize, move or rotate nodes, but has a stronger focus on automatic layout algorithms and applications of [Graph Theory](https://en.wikipedia.org/wiki/Graph_theory). It is suited towards software which requires finer-grained customization of functionality than off-the-shelf packages.
 
-```sh
-$ npm install
-```
 
-To watch the core package, execute
+The `maxGraph` library uses no third-party software, it requires no plugins and can be integrated in virtually any framework (it's vanilla JS).
```

Comment:
This is kept from the original mxGraph README. I suggest we remove it as it doesn't provide valuable information.
```suggestion
```

Reply:
keep for now